src/app/products/products.service.ts

```diff
@@ -42,4 +42,16 @@ export class ProductsService {
     return this.http.get('/fixeddepositproducts');
   }
 
+  /**
+   * @param {any} Fetches Product mixes.
+   * @returns {Observable<any>}
+   */
+  getProductMixes(useHttpParams: boolean = false): Observable<any> {
+    let httpParams = new HttpParams();
+    if (useHttpParams) {

```

Comment:
Remove this, we will always be having http params here as this is specifically for getting product mix data.